docs/online-boutique-asm-manifests/README.md

```diff
@@ -2,14 +2,25 @@
 
 _Note: The [base/all/kubernetes-manifests.yaml](base/all/kubernetes-manifests.yaml) file contains the definition for all the Kubernetes resources required to deploy the Online Boutique (https://github.com/GoogleCloudPlatform/microservices-demo) sample application._
 
+## Quickstart
+
+This sample will deploy the Online Boutique sample application with Kustomize. 
+To get started, run the following command to set the variables:
+
 ```
 NAMESPACE=onlineboutique
 INGRESS_GATEWAY_NAMESPACE=asm-ingress
 INGRESS_GATEWAY_NAME=asm-ingressgateway
+```
+Using `sed`, run the following to set namespace in the kustomize YAML files
+```
 sed -i "s/ONLINEBOUTIQUE_NAMESPACE/${NAMESPACE}/g" base/for-namespace/kustomization.yaml
 sed -i "s/ONLINEBOUTIQUE_NAMESPACE/${NAMESPACE}/g" authorization-policies/for-namespace/kustomization.yaml
 sed -i "s/ONLINEBOUTIQUE_NAMESPACE/${NAMESPACE}/g;s/INGRESS_GATEWAY_NAMESPACE/${INGRESS_GATEWAY_NAMESPACE}/g;s/INGRESS_GATEWAY_NAME/${INGRESS_GATEWAY_NAME}/g" authorization-policies/for-ingress-gateway/kustomization.yaml
 sed -i "s/ONLINEBOUTIQUE_NAMESPACE/${ONLINEBOUTIQUE_NAMESPACE}/g" sidecars/for-namespace/kustomization.yaml
+```
+Lastly run the following to sets the value of the namespace field in the kustomization file, and to apply the resources.
```

Comment:
nit: I guess it should be "to set the value"?

Reply:
eyes of an eagle! addressing, thanks